models/dalec/R/write.configs.dalec.R

```diff
@@ -121,7 +121,7 @@ write.config.DALEC <- function(defaults, trait.values, settings, run.id) {
       sla <- default.param[which(default.param$cmdFlag == "SLA"),"val"] * 1000 #convert SLA to m2/kgC from m2/gC (dalec default)
     }
     
-    IC.pools <- PEcAn.data.land::prepare_pools(IC.path, constants = list(sla = sla))
+    IC.pools <- PEcAn.data.land::prepare.pools(IC.path, constants = list(sla = sla))
```

Comment:
Oh, that was based on this documentation https://pecanproject.github.io/pecan-documentation/coding-style.html#dont-use-shortcuts . I was wondering why so many functions were still _